tests/test_metgen.py

```diff
@@ -455,7 +403,9 @@ def test_no_attempt_to_match_empty_ancillary_files():
 def test_stage_files(m1, m2, m3, test_config):
     granule = metgen.Granule(
         "foo",
-        metgen.Collection("ABCD", 2),
+        CollectionMetadata(
```

Comment:
Seems like this should be a fixture, since it's repeated multiple times times - ?